src/core/api/track_choice_manager.ts

```diff
@@ -968,6 +1015,49 @@ export default class TrackChoiceManager {
 
     recursiveUpdateVideoTrack(0);
   }
+
+  /**
+   * Set the adaptation for stream.
```

Comment:
To me, the TrackChoiceManager doesn't really now what the "stream" is.
It just emits the right track through a Subject that was given to it.